jobs/SprintTag/create_tag.groovy

```diff
@@ -0,0 +1,59 @@
+node{
+    withEnv([
+        "tag_name=${tag_name}",
+        "JUMP_VERSION=${JUMP_VERSION}",
```

Comment:
per @PengTian0 Tian
due to we have decided to "jump version" until the official release-tag added  ( jump was did when rc-tag added).
so the ${JUMP_VERSION} default being FALSE. which was set in Jenkins.